run

```diff
@@ -12,7 +12,7 @@ fi
 options=''
 [[ -n ${SSH_AUTH_SOCK} ]] && options+=" -v ${SSH_AUTH_SOCK}:/run/ssh.sock -e SSH_AUTH_SOCK=/run/ssh.sock"
 
-CACHE_CONTAINER=cache-container-$(pwd | sha1sum | awk '{print $1}')
+CACHE_CONTAINER=cache-container-$(pwd | if [ `uname` == "Darwin" ]; then shasum; else sha1sum; fi | awk '{print $1}')
```

Comment:
shasum is available on Linux as well; at least on Arch. It seems to be missing in a Debian based docker container though. Maybe just check if shasum is available and fallback to sha1sum. We can omit the uname test then.